.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,68 @@
+      label: Steps to reproduce
+      description: |
+        You must provide a clear list of steps and code to reproduce the problem.
+      value: |
+        1. …
+        2. …
+    validations:
+      required: true
+  - type: input
+    id: reproducible-example
+    attributes:
+      label: Reproducible example repository 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)